src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -902,6 +903,30 @@ public static bool IsDefaultExpressionContext(this SyntaxTree syntaxTree, int po
             return false;
         }
 
+        public static bool IsFunctionPointerTypeArgumentContext(
+            this SyntaxTree syntaxTree,
+            int position,
+            SyntaxToken tokenOnLeftOfPosition,
+            CancellationToken cancellationToken)
+        {
+            var token = tokenOnLeftOfPosition;
+            token = token.GetPreviousTokenIfTouchingWord(position);
+
+            switch (token.Kind())
+            {
+                case SyntaxKind.LessThanToken:
+                case SyntaxKind.CommaToken:
+                    return token.Parent is FunctionPointerTypeSyntax;
```

Comment:
the comma is parented direction by the function pointer type?  i.e. we don't have FunctionPointerTypeList?  seems strangely inconsistent with the rest of hte syntactic model.

Reply:
It probably should be a separate parent, I'm planning on changing that as part of the rework.

Reply:
if you have an issue tracking the "missing node" issue, can you potentially add to that?  would prefer tehse not get lost, esp. as we get into crunch time.

Reply:
Added a link.